source/_components/mysensors.markdown

```diff
@@ -171,6 +171,14 @@ void receive(const MyMessage &message) {
 }
 ```
 
+### {% linkable_title Heartbeats %}
+
+Sending a heartbeat from the MySensors sketch to home assistant activates the smartSleep functionality in home assistant. This means that messages are buffered and only sent to the device upon receiving a heartbeat from the device. State changes are stored so that only the last requested state change is sent to the device. Other types of messages are queued in a FIFO queue. SmartSleep is useful if you want to sleep a device, eg to be able to have it battery powered.  See the MySensors library API for information on how to send heartbeats and sleep device.  
```

Comment:
* I think you should write: "Sending a heartbeat from the MySensors device to Home Assistant activates...".
* I think we write home assistant with capital H and A in the docs: Home Assistant. Please check that we're consistent with the rest of the docs.
* SmartSleep is specifically useful for actuator devices, that expects incoming messages. Sensors that only send messages do not really need to be "smartSleeping". Please clarify this.